src/main/java/modtweaker2/mods/appeng/handlers/Inscriber.java

```diff
@@ -83,7 +84,7 @@ public static void addRecipe(IItemStack[] imprintable, IItemStack plateA, IItemS
 
 	public static class Add extends BaseListAddition<IInscriberRecipe> {
 		public Add(IInscriberRecipe recipe) {
-			super(Inscriber.name, AEApi.instance().registries().inscriber().getRecipes());
+			super(Inscriber.name, new ArrayList<>(AEApi.instance().registries().inscriber().getRecipes()));
```

Comment:
What is your intention here? Pretty sure you are not supposed to make a copy of the original list, as otherwise all the addition/removal would operate on this copy and not original list. I doubt if the changes would be visible to the inscriber.

Reply:
I just changed that because Eclipse complained because `getRecipes()` returns a `Collection<IInscriberRecipe>` but super requires a `List<IInscriberRecipe>`